packages/splits-v2/foundry.toml

```diff
@@ -17,6 +17,7 @@ test = "test"
 allow_paths = ['../../node_modules']
 fs_permissions = [{ access = "read-write", path = "./" }]
 evm_version = "shanghai"
+gas_limit = 36_000_000
```

Comment:
do we need to make this change for foundry.toml in smart vaults?

anyway to have like a parent foundry.toml at the root of the repo (so we don't have to remember this setting for new packages as well)? i see there is a foundry.toml at the root, not sure how that's used in each package.